src/main/java/seedu/task/logic/commands/HelpCommand.java

```diff
@@ -1,26 +1,52 @@
 package seedu.task.logic.commands;
 
-
 import seedu.task.commons.events.ui.ShowHelpEvent;
 import seedu.taskcommons.core.EventsCenter;
+//import java.util.regex.Matcher;
+//import java.util.regex.Pattern;
+
+import javafx.scene.control.TextField;
 
 /**
  * Format full help instructions for every command for display.
+ * 
+ * @author Poh Yee Heng
  */
 public class HelpCommand extends Command {
+	public final String commandWord;
```

Comment:
you mean showing_help_clear, showing_help_exit and showing_help_message ? 
like all three of them ?